site2/website/blog/2022-07-15-pulsar-summit-san-francisco.md

```diff
@@ -0,0 +1,53 @@
+---
+author: Alice Bi, Karin Landers
+title: "Speakers and Sponsorship Prospectus Announced for Pulsar Summit San Francisco 2022"
+date: 2022-07-15
+---
+
```

Comment:
There needs to a link to the pulsar-summit.org website's event page, the speaker list, and the schedule.

Reply:
Thanks. I added these links to the text.